mozilla_schema_generator/probes.py

```diff
@@ -93,6 +100,12 @@ def _set_dates(self, first_added_value: dict):
         self.first_added = min(vals)
         self.last_change = max(vals)
 
+    def _set_description(self, definition):
+        if "description" in definition:
+            self.description = definition["description"]
```

Comment:
Looks like this is set both here and in `super().__init__`. I think we can remove the latter and just set it here.